android/Tests/Chirality/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I think perhaps we should make this `Chirality` test app a harness for various keyboards. So call it instead `KeyboardHarness` or something like that? Because we don't want to boilerplate all that Java app code just to add another keyboard...

Reply:
yeah, I can rename the test app to `KeyboardHarness`. 
@jahorton asks if we also plan to have a CI build of the test app. 

Also note, the app is copied off of KMSample1, so some features are missing (like using system keyboard ala KMSample2)

Reply:
At some point we should have a CI build for the test harness app. This app should be targeting keyboard functionality and bug repro with keyboards rather than integration functionality, I think. So not having system keyboard support is fine.